docs/reference/esql/metadata-fields.asciidoc

```diff
@@ -20,6 +20,10 @@ supported ones are:
   * <<mapping-ignored-field,`_ignored`>>: the ignored source document fields. The field is of the type
   <<keyword,keyword>>.
 
+  * experimental::["scoring is experimental and only available in SNAPSHOT versions."]
```

Comment:
because this is snapshot only for now, I think we want to hold off on updating the docs.